### ISSUE TITLE
Interpolate and/or spline methods

### ISSUE THREAD

**Issue by djsegal** [JuliaPy/SymPy.jl#179]
I see you expose the interpolate method from the SymPy library, but I can't get it to work.

Have you had any luck with it?

**Comment by jverzani** [comment]
Well... no. That method has the wrong signature for the metaprogramming trick. I'll need to wrap it separately. Until then, these should work:

```
@vars x
sympy_meth(:interpolate, [1,2,5], x)
sympy_meth(:interpolate, [(1,1), (2,2), (3,5)], x)
```

I don't really like the latter calling method. In `Polynomials` these are specified as `xs, ys`. Any thoughts on what should be done there?

**Comment by jverzani** [comment]
cf #180 if you wish to make a comment on the interface.

**Comment by jverzani** [comment]
Closed by #180
